catalog/spatial/registry/registry-federation-admin-service/src/main/java/org/codice/ddf/registry/federationadmin/service/FederationAdminService.java

```diff
@@ -14,42 +14,223 @@
 package org.codice.ddf.registry.federationadmin.service;
 
 import java.util.List;
+import java.util.Set;
 
 import ddf.catalog.data.Metacard;
-import ddf.catalog.federation.FederationException;
-import ddf.catalog.source.IngestException;
-import ddf.catalog.source.SourceUnavailableException;
-import ddf.catalog.source.UnsupportedQueryException;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryPackageType;
 
 /**
  * Service that provides the interface to get and add registry metacards
+ * <b> This code is experimental. While this interface is functional and tested, it may change or be removed in a future version of the library. </b>
```

Comment:
I think it could probably be removed when we are at the point to enable registry by default. I figure by then we have most of the issues worked out.